src/components/form-draft/testing.vue

```diff
@@ -147,7 +166,15 @@ export default {
     ],
     // This text will be shown in ODK Collect when testing a Draft Form. {name}
     // is the title of the Draft Form.
-    "collectProjectName": "[Draft] {name}"
+    "collectProjectName": "[Draft] {name}",
+    "datasetsPreview": {
+      "title": "Datasets Preview",
+      "body": [
+        "This form can add entities to a dataset.",
+        "In this preview release of Datasets, you must publish your Form to see the Dataset update. In a future release of Central, you will be able to test Dataset functionality in a draft state.",
```

Comment:
These two strings differ from the body of the release criteria just in capitalization I think.

Reply:
Capitalization looks good except for final "Draft".